packages/typescript/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
+  "compilerOptions": {
+    "strictNullChecks": true
```

Comment:
This is now required by `@typescript-eslint/eslint-plugin` when type checking is enabled.